plugin/core/rpc.py

```diff
@@ -43,7 +43,7 @@ def get_project_path(self):
         return self.project_path
 
     def has_capability(self, capability):
-        return capability in self.capabilities
+        return capability in self.capabilities and self.capabilities[capability] is not False
```

Comment:
Perhaps is `return self.capabilities.get(capability, False)` is slightly less verbose?

Reply:
@rwols that syntax reads better but eg `self.capabilities.get('completionProvider', False)` would return a dict with options instead of a boolean.